.github/workflows/build-uc2.yml

```diff
@@ -35,7 +35,7 @@ jobs:
               shared: 'yes',
               mingw: MINGW64,
               mingw-arch: x86_64,
-              artifact: 'windows_mingw64.7z',
```

Comment:
Reason for this change?

Reply:
The static build overrides the shared build.
Making it difficult to test one of the builds first (shared)

Reply:
@wtdcode, In shared I got a problem when using standalone libunicorn.dll. Requires msys2 libs as:
* `libgcc_s_seh-1.dll`,
* `libwinpthread.dll`

Suggest: add `-static-libgcc` or `-static` flag to MinGW-shared.

Reply:
> @wtdcode, In shared I got a problem when using standalone libunicorn.dll. Requires msys2 libs as:
> 
> * `libgcc_s_seh-1.dll`,
> * `libwinpthread.dll`
> 
> Suggest: add `-static-libgcc` or `-static` flag to MinGW-shared.

Just adding this to your PR should be fine?

Reply:
new commit